docs/tutorials/10_effective_dimension.ipynb

```diff
@@ -0,0 +1,763 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# plot the normalized effective dimension for the model\n",
+    "plt.plot(n, np.array(global_eff_dim_1) / d)\n",
+    "plt.xlabel(\"Number of data\")\n",
+    "plt.ylabel(\"Normalized GLOBAL effective dimension\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## 4. Local Effective Dimension Example\n",
+    "As explained in the introduction, the local effective dimension algorithm only uses **one** set of weights. The `LocalEffectiveDimension` class enforces this constraint to ensure that these calculations are conceptually separate, but the rest of the implementation is shared with the `EffectiveDimension` class.\n",
+    "\n",
+    "Let's see how the `LocalEffectiveDimension` changes when we train a QNN for classification.\n",
+    "\n",
+    "First, we create a classification dataset, similarly to what you may see in other tutorials."
```

Comment:
Does this `similarly to what you may see in other tutorials` add any value here. There is no specific link where perhaps I might gain further insight; or is it trying to say its not uncommon to 'make up' i.e. create a dataset, others do it